apps/spa/src/AppLayout.tsx

```diff
@@ -14,15 +16,37 @@ function AppLayout(): JSX.Element {
     }
   }, [pathname]);
 
+  const handleAuthRedirectCallback = (appState?: AppState) => {
+    if (appState?.returnTo) {
+      navigate(appState.returnTo);
+      return;
+    }
+
+    // Removes the code and state query params
+    navigate("/home");
+  };
+
   return (
-    <div>
+    <Auth0Provider
+      domain={import.meta.env.VITE_AUTH0_DOMAIN}
+      clientId={import.meta.env.VITE_AUTH0_CLIENT_ID}
+      authorizationParams={{
+        redirect_uri: window.location.href,
+        audience: import.meta.env.VITE_AUTH0_AUDIENCE,
+        scope: "openid profile read:all_data",
```

Comment:
Perhap `read:all_data` needs to be in the `profile-service` package. Consumers of the profile service should be able to look up what scopes are available to them.